lab_3_keywords_textrank/main.py

```diff
@@ -564,7 +662,7 @@ def get_top_keywords(self, n_keywords: int) -> tuple[int, ...]:
             tuple[int, ...]
                 top n most important tokens in the encoded text
         """
-        pass
+        return tuple(sorted(sorted(self._scores), key=lambda x: self._scores[x], reverse=True))[:n_keywords]
```

Comment:
Also dictionaries are ordered, it is a byproduct of development, and using it is considered a bad practice

Use `key` argument to establish a secondary sorting order